docs/guide/running-extern-robot-controllers.md

```diff
@@ -38,10 +38,10 @@ Generic Webots environment variables needed for all the controller languages:
 
 %tab "Linux"
 
-| Environment Variable     | Typical Value                                    |
-|--------------------------|--------------------------------------------------|
-| WEBOTS\_HOME             | `/usr/local/webots`                              |
-| LD\_LIBRARY\_PATH        | add `${WEBOTS_HOME}/lib/controller`              |
+| Environment Variable                                  | Typical Value                                    |
+|-------------------------------------------------------|--------------------------------------------------|
+| WEBOTS\_HOME                                          | `/usr/local/webots`                              |
+| LD\_LIBRARY\_PATH (not needed for Python controllers) | add `${WEBOTS_HOME}/lib/controller`              |
```

Comment:
That applies from Python 3.8 on:
```suggestion
| LD\_LIBRARY\_PATH (not needed for Python >= 3.8 controllers) | add `${WEBOTS_HOME}/lib/controller`              |
```
And it also applies to Windows and macOS.
Moreover, for Windows, we should also document the `MSYS2_HOME` environment variable in case of development environment.

Reply:
Why does it apply only to Python 3.8? It should work with any version.

`rpath` is not supported for Windows and macOS, so it will not work there.

Reply:
Because os.add_dll_directory() was introduced only in this version of Python, see: https://github.com/cyberbotics/webots/blob/master/src/controller/python/controller.i#L28-L37

Reply:
See the explanation here: https://stackoverflow.com/questions/60509120/python-3-8-not-loading-c-dll

Reply:
And that is specific to Windows: https://github.com/Toblerity/Fiona/issues/851

Reply:
https://docs.python.org/3/library/os.html#os.add_dll_directory

Reply:
Oh nice, since Windows doesn't support $ORIGIN this is great. Let me try to make $ORIGIN work for macOS, so we have all platforms covered.

Reply:
It indeed works on macOS as well:
```
lukic@Cyberboticss-Mac-mini webots % echo $DYLD_LIBRARY_PATH

lukic@Cyberboticss-Mac-mini webots % python3.7 -c "import controller; controller.Supervisor()"
Webots doesn't seems to be ready yet. Retrying in 1 second.
...
```